web/src/lane-editor/App.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+  import init from "osm2streets-js";
+  import { onMount } from "svelte";
+  import AppSwitcher from "../AppSwitcher.svelte";
+  import ImportControls from "../osm2streets-svelte/import/ImportControls.svelte";
+  import MainLayers from "../osm2streets-svelte/layers/MainLayers.svelte";
+  import Layout from "../osm2streets-svelte/Layout.svelte";
+  import Map from "../osm2streets-svelte/Map.svelte";
+  import EditWayControls from "./EditWayControls.svelte";
+
+  onMount(async () => {
+    await init();
+  });
+</script>
+
+<Layout>
```

Comment:
Svelte is component based, so we can very naturally express the overall organization of the app and share components so much more easily than with vanilla JS